code/modules/spells/vampire_helpers.dm

```diff
@@ -89,10 +89,10 @@
 		var/datum/role/vampire/V = M.GetRole(VAMPIRE)
 		var/obj/item/weapon/nullrod/N = locate(/obj/item/weapon/nullrod) in get_contents_in_object(src)
 		if (N)
-			if (/datum/power/vampire/undying in V.current_powers)
+			if (locate(/datum/power/vampire/undying) in V.current_powers)
 				to_chat(M.current, "<span class='warning'>An holy artifact has turned our powers against us!</span>")
 				return VAMP_FAILURE
-			if (/datum/power/vampire/jaunt in V.current_powers)
+			if (locate(/datum/power/vampire/jaunt) in V.current_powers)
 				to_chat(M.current, "<span class='warning'>An holy artifact protects [src]!</span>")
```

Comment:
*A holy

Reply:
Not necessarily. In British English, words starting with "h" are treated as if they had a vowel instead. While weird to US English speakers, it's not grammatically wrong.

Also, OOC: Thanks, now I know my target is british

Reply:
An 'oly